### PR TITLE
Add Security Dashboard link to Nav

### PR DESCRIPTION
**Pull request by joeclayallday**
This PR adds a sidebar nav link that points to `https://dashboard.aptible.com/controls` in deploy-ui. Users must log into deploy-ui to see the page.

Screen Shot 2023-08-02 at 3 38 44 PM

### CODE REVIEW

File: src/ui/shared/application-sidebar.tsx [hunk 1]
```diff
@@ -44,6 +46,7 @@ export const ApplicationSidebar = () => {
     { name: "Databases", to: databaseUrl(), icon: <IconCylinder /> },
     { name: "Deployments", to: deploymentsUrl(), icon: <IconCloud /> },
     { name: "Activity", to: activityUrl(), icon: <IconHeart /> },
+    { name: "Security & Compliance", to: securityDashboardUrl(), icon: <IconShield /> },
```
Comment (neurosnap) [inline src/ui/shared/application-sidebar.tsx:49]: I think we want to link directly to dashboard https://dashboard.aptible.com/controls

File: src/routes/index.ts [hunk 2]
```diff
@@ -226,3 +226,6 @@ export const stackDetailVpcPeeringsUrl = (id: string) =>
 
 export const SEARCH_PATH = "/search";
 export const searchUrl = () => SEARCH_PATH;
+
+export const SECURITY_DASHBOARD_PATH = "/search";
+export const securityDashboardUrl = () => SECURITY_DASHBOARD_PATH;
```
Comment (neurosnap) [inline src/routes/index.ts:231]: I don't think we'll need these as we want to link directly to deploy-ui

**Review by neurosnap — CHANGES_REQUESTED** [review]

File: src/routes/index.ts (hunk 2 above)
Comment (joeclayallday) [reply src/routes/index.ts]: Thanks!

File: src/ui/shared/application-sidebar.tsx (hunk 1 above)
Comment (joeclayallday) [reply src/ui/shared/application-sidebar.tsx:49]: Thanks!

File: src/routes/index.ts [hunk 3]
```diff
@@ -226,3 +226,6 @@ export const stackDetailVpcPeeringsUrl = (id: string) =>
 
 export const SEARCH_PATH = "/search";
 export const searchUrl = () => SEARCH_PATH;
+
+export const securityDashboardUrl = () =>
+  "https://dashboard.aptible.com/controls";
```
Comment (madhuravius) [inline src/routes/index.ts:231]: not sure if you want to do this but there's a slightly different value by env and we could make it easier for staging envs to go to their appropriate link (not that it matters too much for now)

**Review by madhuravius — COMMENTED** [review]
lgtm, i think we could update that link for staging though if we wanted to (probably not worth the effort as we don't really establish staging security dashboards in most staging envs, right @neurosnap ?)

**Review by neurosnap — APPROVED** [review]

**Review by madhuravius — APPROVED** [review]

**Pull request merged**